keyboards/owlab/link/hotswap/readme.md

```diff
@@ -0,0 +1,21 @@
+# LINK - hotswap PCB
+
+![LINK](https://imgur.com/a/Jqcl5Dq.png)
```

Comment:
This uses a smaller, but still hi res image, that is more suitable for github markdown rendering. 
```suggestion
![LINK](https://imgur.com/a/Jqcl5Dqh.png)
```